readme.md

```diff
@@ -218,29 +220,44 @@ Contains libraries that enable interactions with the Device Proviosining service
 Contains libraries that provide modeling and JSON serialization capabilities on top of the raw messaging library.
 
 
-# Long Term Support
+# Releases
 
-The project offers a Long Term Support (LTS) version to allow users that do not need the latest features to be shielded from unwanted changes.
+The C SDK offers releases for new features, security updates, and Long Term Support (LTS). General bug fixes will not receive a separate release. Versioning follows [semantic versioning](https://semver.org/), `x.y.z.` or `major.minor.patch`. Any time the version is updated, it will be tagged `x.y.z`.
 
-A new LTS version will be created every 6 months. The lifetime of an LTS branch is currently planned for one year. LTS branches receive all bug fixes that fall in one of these categories:
+## Features and security updates
 
-- security bugfixes
-- critical bugfixes
+New features and security updates will be released on the main branch. These releases will be tagged using the date formatted `yyyy-mm-dd`. A feature release will bump the `minor` version and reset the `patch` version to 0. A security update will bump the `patch` version only.
 
-No new features will be picked up in an LTS branch.
+## Long Term Support
 
-LTS branches are named lts_*mm*_*yyyy*, where *mm* and *yyyy* are the month and year when the branch was created. An example of such a branch is *lts_07_2017*.
+New LTS releases branch off of main every six months and will be tagged `LTS_<mm_yyyy>_Ref01`. A new LTS release will inherit the existing version from the main branch. LTS branches are named `lts_mm_yyyy` for the month and year the branch was created. The lifetime of an LTS branch is one year, afterwhich the branch is deleted.
```

Comment:
"inherit the version from the main branch at the time of the release." ?

Reply:
I would suggest not embedding the lifetime or frequency of the releases in the text like this.
It better be all in the table we keep, and you can add a reference in the description here for the frequency and lifetime. It will be more consistent, and easier to maintain in the future.

Reply:
I'm confused, as your comment suggests adding a reference to frequency and lifetime but also says not to include it.  Can you please clarify for me what you mean? Also, lifetime and frequency were mentioned prior to my change.

Per the "inherit" bit, do you mean to use what you posted in place of what I have on line 233, or are you asking why I added in that info?

Reply:
```suggestion
New LTS releases branch off of main every six months and will be tagged `LTS_<mm_yyyy>_Ref01`. A new LTS release will inherit the version from the main branch at the time of the release. LTS branches are named `lts_mm_yyyy` for the month and year the branch was created.
```

Reply:
I see, just keep those specifics in the table.